### ISSUE TITLE
Migrate documentation from Tizen.org

### ISSUE THREAD

**Issue by kad** [01org/bmap-tools#7]

**Comment by kad** [comment]
@dbkinder would you have time to help a bit with documentation move ?

**Comment by dbkinder** [comment]
The tizen documentation for bmaptool is 2+ years old and may need a good technical review.  Did you want to move these doc pages to this github repo's wiki: https://github.com/01org/bmap-tools/wiki ?

**Comment by kad** [comment]
Yes, I wanted it to move here, on github wiki and cleanup majority of Tizen specifics. Just to keep documentation generic and simple.

**Comment by dbkinder** [comment]
If you really wanted to keep this alive, the docs are still available at http://web.archive.org/web/20190803155449/https://source.tizen.org/documentation/reference/bmaptool
